samples/boards/rpi_pico/uart_pio/prj.conf

```diff
@@ -0,0 +1 @@
+# nothing here
```

Comment:
if nothing, just delete the file?

Reply:
prj.conf files are required even if empty